driver/src/orderbook/streamed/balance.rs

```diff
@@ -53,7 +53,7 @@ impl Balance {
         Ok(())
     }
 
-    pub fn get_balance(&self, batch_id: BatchId) -> BigInt {
+    pub fn get_balance_at_beginning_of_batch(&self, batch_id: BatchId) -> BigInt {
```

Comment:
Nice, I think the method renames helps clarify exactly what balance is being returned